### PR TITLE
Add optional zerolog hook to output log location in format compatible with GCP logging.

### PR DESCRIPTION
**Pull request by vadimberezniker**
---

**Version bump**: Patch

### CODE REVIEW

File: server/util/log/log.go [hunk 1]
```diff
@@ -111,24 +117,46 @@ func StructuredLogger() zerolog.Logger {
...
+}
+
+func Configure(opts Opts) error {
 	var logger zerolog.Logger
-	if enableStructured {
+	if opts.EnableStructured {
 		logger = StructuredLogger()
 	} else {
-		logger = LocalLogger(level)
+		logger = LocalLogger(opts.Level)
```
Comment (tylerwilliams) [inline server/util/log/log.go:143]: I don't think LocalLogger needs level anymore -- can you remove the need for this arg?

**Review by tylerwilliams — APPROVED** [review]
LGTM

**Review by tylerwilliams — COMMENTED** [review]
Looks like you might need to get the rbe binary too?

File: server/util/log/log.go (hunk 1 above)
Comment (vadimberezniker) [reply server/util/log/log.go]: Done.

**Pull request merged**